src/Agent.Worker/ContainerOperationProvider.cs

```diff
@@ -32,7 +35,7 @@ public override void Initialize(IHostContext hostContext)
             _dockerManger = HostContext.GetService<IDockerCommandManager>();
         }
 
-        public async Task StartContainerAsync(IExecutionContext executionContext, object data)
+        private async Task StartContainerAsync(IExecutionContext executionContext, object data)
```

Comment:
`ContainerInfo container` instead of `object data`

Reply:
Done